README.md

```diff
@@ -62,6 +62,7 @@ Send pull request if you want to be listed here.
 * EDNS0, NSID;
 * AXFR/IXFR;
 * TSIG, SIG(0);
+* DNS over TLS;
```

Comment:
Think we can make some more nice about this feature in the README :) It's a completely new transport...